content/docs/components-and-props.md

```diff
@@ -1,262 +1,262 @@
 ---
 id: components-and-props
-title: Components and Props
+title: Components i propietats
 permalink: docs/components-and-props.html
 redirect_from:
-  - "docs/reusable-components.html"
-  - "docs/reusable-components-zh-CN.html"
-  - "docs/transferring-props.html"
-  - "docs/transferring-props-it-IT.html"
-  - "docs/transferring-props-ja-JP.html"
-  - "docs/transferring-props-ko-KR.html"
-  - "docs/transferring-props-zh-CN.html"
-  - "tips/props-in-getInitialState-as-anti-pattern.html"
-  - "tips/communicate-between-components.html"
+- "docs/reusable-components.html"
```

Comment:
Review all of these lines after this one. These should be the same as in the original source.